bootstrap/021_external_functions.sql

```diff
@@ -156,3 +201,8 @@ BEGIN
       INSERT (key, value)
       VALUES (source.key, source.value);
 END;
+
+CREATE OR REPLACE PROCEDURE admin.setup_external_functions() RETURNS STRING LANGUAGE SQL AS
+BEGIN
+    call admin.setup_external_functions('reference(\'opscenter_api_integration\')');
+END;
```

Comment:
I'm not convinced this will do what I want it to do, but I'm trying to fix the `devdeploy.py` setup where you have to create your own API integration via the scripting block.

These changes work for me to set my own api integration name and then pass that into the `setup_external_functions` procedure, but I fear I broke the `reference('opscenter_api_integration')` path through deploy.py.